.travis.yml

```diff
@@ -7,7 +7,12 @@ language: c++
 # TODO(2019-07-21): Add "ffmpeg=1" if FFmpeg 4.x becomes available in Ubuntu
 env:
   global:
-    - COMMON_FLAGS="-j4 test=1 mad=1 faad=1 opus=1 modplug=1 wv=1 hss1394=0 virtualize=0 debug_assertions_fatal=1 verbose=0"
+    # For SCons builds
+    - SCONSFLAGS="test=1 mad=1 faad=1 opus=1 modplug=1 wv=1 hss1394=0 virtualize=0 debug_assertions_fatal=1 verbose=0"
```

Comment:
I would suggest to align the naming:
SCONS_BUILD_FLAGS
CMAKE_BUILD_COMMON_FLAGS
CMAKE_BUILD_EXTRA_FLAGS

If we ever need separate COMMON and EXTRA flags for SCons we might apply the same split as for CMake.

Reply:
I'd like to keep `SCONSFLAGS`, because it is read by `scons` automatically (so we don't need to specify it when invoking it).

How about renaming the other variables to `CMAKEFLAGS`/`CMAKEFLAGS_EXTRA`?

Reply:
I wasn't aware of this fact. Sounds like a plan :+1: